unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/WebRequest/Interfaces/IWebRequestController.cs

```diff
@@ -41,6 +42,27 @@ public interface IWebRequestController
             int timeout = 0,
             bool disposeOnCompleted = true);
 
+        /// <summary>
+        /// Download data from a url with specific headers.
+        /// </summary>
+        /// <param name="url">Url where to make the request.</param>
+        /// <param name="headers">Headers for the request</param>
+        /// <param name="downloadHandler">Downloader handler to be used by the GET request.</param>
+        /// <param name="OnSuccess">This action will be executed if the request successfully finishes and it includes the request with the data downloaded.</param>
+        /// <param name="OnFail">This action will be executed if the request fails.</param>
+        /// <param name="requestAttemps">Number of attemps for re-trying failed requests.</param>
+        /// <param name="timeout">Sets the request to attempt to abort after the configured number of seconds have passed (0 = no timeout).</param>
+        /// <param name="disposeOnCompleted">Set to true for disposing the request just after it has been completed.</param>
+        WebRequestAsyncOperation Get(
+            string url,
+            Dictionary<string, string> headers,
+            DownloadHandler downloadHandler = null,
+            Action<UnityWebRequest> OnSuccess = null,
+            Action<UnityWebRequest> OnFail = null,
+            int requestAttemps = 3,
+            int timeout = 0,
+            bool disposeOnCompleted = true);
```

Comment:
Could we unify this in only one `Get`? Both signatures are identicals except the `headers` param, so we can add it as optional at the end and we would avoid to have 2 methods almost equals. Thanks!